obspy/io/sac/tests/test_sactrace.py

```diff
@@ -0,0 +1,187 @@
+import os
+import unittest
+import datetime
+
+import numpy as np
+
+from future.utils import native_str
+
+from obspy import UTCDateTime
+from obspy.core.util import NamedTemporaryFile
+
+from ..sactrace import SACTrace
+
+class SACTraceTestCase(unittest.TestCase):
+    """
+    Test suite for obspy.io.sac.sactrace
+    """
+    def setUp(self):
+        self.path = os.path.dirname(__file__)
+        self.file = os.path.join(self.path, 'data', 'test.sac')
+        self.filexy = os.path.join(self.path, 'data', 'testxy.sac')
+        self.filebe = os.path.join(self.path, 'data', 'test.sac.swap')
+        self.fileseis = os.path.join(self.path, 'data', 'seism.sac')
+        self.testdata = np.array(
+            [-8.74227766e-08, -3.09016973e-01,
+             -5.87785363e-01, -8.09017122e-01, -9.51056600e-01,
+             -1.00000000e+00, -9.51056302e-01, -8.09016585e-01,
+             -5.87784529e-01, -3.09016049e-01], dtype=np.float32)
+
+    def test_read_binary(self):
+        """
+        Tests for SACTrace binary file read
+        """
+        sac = SACTrace.read(self.file, byteorder='little')
+        self.assertEqual(sac.npts, 100)
+        self.assertEqual(sac.kstnm, 'STA')
+        self.assertEqual(sac.delta, 1.0)
+        self.assertEqual(sac.kcmpnm, 'Q')
+        self.assertEqual(sac.reftime.datetime, datetime.datetime(1978, 7, 18, 8, 0))
+        self.assertEqual(sac.nvhdr, 6)
+        self.assertEqual(sac.b, 10.0)
+        self.assertAlmostEqual(sac.depmen, 9.0599059e-8)
+        np.testing.assert_array_almost_equal(self.testdata[0:10],
```

Comment:
I forget what version of NumPy we updated master to use. Are we allowed to use this test, @krischer?